test/views/MentionsAndSentiments/test.index.js

```diff
@@ -0,0 +1,130 @@
+import React from 'react';
+import assert from 'assert';
+import { shallow } from 'enzyme';
+import Accordion from '../../../views/Accordion/index.jsx';
+import MentionsAndSentiments from '../../../views/MentionsAndSentiments/index.jsx';
+
+describe('mentions_sort_test', () => {
```

Comment:
could it be called `describe('<MentionsAndSentiments />', () => {` ? that way if we decide to add more tests later we don't have to rename it